dracula.conf

```diff
@@ -25,3 +27,20 @@ color6     #8BE9FD
 color14    #9AEDFE
 color7     #BFBFBF
 color15    #E6E6E6
+
+# URL styles
+url_color #BD93F9
+url_style single
+
+# Cursor styles
+cursor #E6E6E6
+cursor_shape beam
```

Comment:
I don't think this is necessarily part of dracula theme? Otherwise everything else looks good to me.

Reply:
```suggestion
```

Reply:
Not necessarily part of the theme, but it looks way better that way :)

I went ahead and removed it.